packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -47,13 +47,12 @@ export const Tab = React.forwardRef<HTMLElement, TabProps>(
       'data-testid': selectors.components.Tab.title(label),
       ...otherProps,
       onClick: onChangeTab,
-      role: 'tab',
       'aria-selected': active,
     };
 
     if (href) {
       return (
-        <div className={tabsStyles.item}>
+        <div className={tabsStyles.item} role="tab">
```

Comment:
I am thinking about removing the wrapping divs here, because it makes more sense semantically. This means the styling might look a bit different.